src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,157 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size = 0;
+    private MyLinkedList.Node<T> first;
+    private MyLinkedList.Node<T> last;
+
+    MyLinkedList.Node<T> node(int index) {
```

Comment:
```suggestion
    private Node<T> getNodeByIndex(int index) {
```
all private methods must be after publics